.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
OMG!, @deschantkn, What is this?

Reply:
Hello @eliemugenzi, are you referring you to the `.netlify` folder?

Reply:
Yes please.

Reply:
It is a folder created by netlify when I tried to deploy there. Since we are no longer using netlify I will delete the folder.